Riot/Modules/Application/AppCoordinator.swift

```diff
@@ -49,7 +49,7 @@ final class AppCoordinator: NSObject, AppCoordinatorType {
     private weak var splitViewCoordinator: SplitViewCoordinatorType?
     fileprivate weak var sideMenuCoordinator: SideMenuCoordinatorType?
     
-    private let userSessionsService: UserSessionsService
+    let userSessionsService: UserSessionsService
```

Comment:
We should try to keep it private and inject this instance from Coordinators to other Coordinators.
For places where we still can't inject instances such as `UserSessionsService` from side to side. The parent does not have the reference or the flow is started from ObjC class.
We can temporary define and use a shared instance like `UserSessionsService.shared` but we still continue to inject it from the outside like: `CoordinatorParameters(userSessionsService: UserSessionsService.shared)` to avoid having singleton inside Coordinators. And in `AppCoordinator.init` we can temporary use `self.userSessionsService = UserSessionsService.shared` until we can avoid singleton in the app.